modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -1967,13 +1967,42 @@ abstract class RunTestDefinitions
     }
   }
 
-  if (!actualScalaVersion.contains("RC"))
-    test("offline mode should fail on missing artifacts") {
+  if (!actualScalaVersion.contains("RC")) {
+    val actualAnnouncedScalaVersion = actualScalaVersion match {
+      case _
+          if actualScalaVersion == Constants.scala3Next &&
+          Constants.scala3Next != Constants.scala3NextAnnounced =>
```

Comment:
Alternatively, we could do `assume( Constants.scala3Next == Constants.scala3NextAnnounced)` inside the tests, if they are different then the tests will be skipped